modules/test/playwright/pages/object-web/ObjectDefinitionsPage.ts

```diff
@@ -60,6 +70,13 @@ export class ObjectDefinitionsPage {
 		return response.json();
 	}
 
+	async deleteObjectFolder(objectFolderName: string) {
+		await this.objectFolderDeleteFolderOption.click();
+		await this.confirmFolderNameInput.click();
+		await this.confirmFolderNameInput.fill(objectFolderName);
```

Comment:
We may need to use object prefix, to avoid code without object prefix and using object prefix on the same line